fe/projects/Seller.Portal/src/shared/helpers/forms/useForm.js

```diff
@@ -134,8 +134,15 @@ function useForm(
 
       setIsDirty(true);
 
+      let value = event.target.value;
+      const fieldType = event.target.type;
+
+      if (parseFloat(value) && fieldType == "number") {
```

Comment:
**suggestion:** Imo parseFloat is redundant here

Reply:
This parseFloat() is to check value variable is possible to parse to number.